src/app/chat/core.nim

```diff
@@ -1,10 +1,18 @@
 import NimQml
 import ../../status/chat as status_chat
 import view
+import messages
 import ../signals/types
 
-var sendMessage = proc (msg: string): string =
-  echo "sending public message"
+var sendMessage = proc (view: ChatsView, msg: string): string =
+  echo "sending public message!"
+  let chatMessage = newChatMessage()
+  chatMessage.userName = "myself"
```

Comment:
The design said `You` and even though the design is very WIP, I do prefer `You` to `myself`